data-serving/data-service/test/controllers/case.test.ts

```diff
@@ -206,3 +208,44 @@ describe('DELETE', () => {
             .expect(404);
     });
 });
+
+describe('Sources', () => {
+    it('are created if needs be', async () => {
+        const c = await new Case(minimalCase).save();
+        expect(c.sources).toHaveLength(1);
+    });
+    it('are reused if existing on create', async () => {
+        const source = await new Source({ url: 'cdc.gov' }).save();
```

Comment:
Might be clearer to use `minimalCase.url`?

Reply:
good point, done.